subworkflows/local/db_check.nf

```diff
@@ -17,10 +18,22 @@ workflow DB_CHECK {
         .dump(tag: "db_split_csv_out")
         .map { create_db_channels(it) }
         .dump(tag: "db_channel_prepped")
-        .set{ dbs }
+
+    parsed_samplesheet
+        .branch {
+            untar: it[1].toString().endsWith(".tar.gz")
```

Comment:
Should we also allow uncompressed input (`.tar` in addition to `.tar.gz`)?

Reply:
Yes, probably. But I think we should only add stuff when needed otherwise we could end up with a lot of pollution (.gz, .zip, .bzw, .xz.....)